TestGenerator

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+echo "how many tests would you like to run ?"
+read NUM_OF_TESTS
+echo "Generating ${NUM_OF_TESTS} Tests..."
+for (( i=1; i<=${NUM_OF_TESTS};i++ ))
+do
+	echo -en "\r$i"
+	python3.6 generate_test_file.py>>winner.txt
```

Comment:
Make python3.6 a variable so people can easily change it from the beginning of the code.